### PR TITLE
Added Settings variable into the default handler

### PR DESCRIPTION
**Pull request by bouiboui**
It will be used most of the time, we win a couple of seconds each time with this line.

### CODE REVIEW

File: generators/app/templates/src/handlers/foo.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const debug = require('debug')('API');
 const logger = require('../helpers/logger');
 const LambdaWrapper = require('../helpers/lambdawrapper');
 const <%= classBaseName %>Service = require('../libs/<%= routeName %>/<%= routeName %>.service');
+const Settings = require('../../config/env.json');
```
Comment (BasileTrujillo) [inline generators/app/templates/src/handlers/foo.js]: Settings is not used! Eslint is crying!

Comment (bouiboui) [inline generators/app/templates/src/handlers/foo.js]: Fixed.

File: generators/app/templates/src/handlers/foo.js [hunk 2]
```diff
@@ -16,7 +17,7 @@ module.exports.run = (event, context, callback) => {
   const service = new <%= classBaseName %>Service(true);
 
   // Each run is a chained promise
-  app.run(() => service.setBar(false))
+  app.run(() => service.setBar(Settings.<%= routeName %>.bar_value))
```
Comment (BasileTrujillo) [inline generators/app/templates/src/handlers/foo.js]: Nice, but now can you put this sample value inside `env.dev.json`?
Actually the generator copy this file to `env.json`.

Comment (BasileTrujillo) [inline generators/app/templates/src/handlers/foo.js]: and without <%= routeName %> it seems to be good

Comment (bouiboui) [inline generators/app/templates/src/handlers/foo.js]: No I can't actually, that's a mess to implement. I will change this to something else.

**Comment by bouiboui** [comment]
@tomexa Can you merge?

**Pull request merged**